Documentation/usingupm.md

```diff
@@ -0,0 +1,77 @@
+# Mixed Reality Toolkit and Unity Package Manager
+
+Starting with version 2.5.0, the Microsoft Mixed Reality Toolkit is available using the Unity Package Manager (UPM), on Unity 2019.4 and newer. Using UPM, the MRTK gains dependency management and a simple update path, as will be described in the following sections.
+
+## Installing Mixed Reality features using the Unity Package Manager
```

Comment:
Do we also want to upload the .tgz files to our releases, so people can download them and import to UPM via the [local tarball option](https://docs.unity3d.com/Manual/upm-ui-tarball.html)?

Reply:
possibly

Reply:
let's add those instructions when we decide if that is the route we decide to take